css/style.css

```diff
@@ -1,7 +1,355 @@
+  color: #EAEAEA;
+}
+
+.nav__button {
+  background-color: #171E25;
+  border-color: #171E25;
+}
+
+.nav__button:hover {
+  background-color: #73A4B8;
+}
+
+.nav__button--vis:first-child:hover {
+  box-shadow: 0 -2px #6FC3E5;
+}
+
+.nav__button--vis:last-child:hover {
+  box-shadow: 0 2px #6FC3E5;
+}
+
+.nav__button--vis:first-child {
+  border-radius: 3px 3px 0 0;
+}
+
+.nav__button--vis:last-child {
+  border-radius: 0 0 3px 3px;
+  border-top: 2px solid #12181B;
+  margin-top: 0;
+}
+
+.nav__button--data:hover {
+  box-shadow: 0 -2px #6FC3E5, 0 2px #6FC3E5;
+}
+
+.nav__links {
+  margin-top: 20px;
+  text-align: center;
+}
+
+.nav__links .icon {
+  margin-right: 0;
+}
+
+/* ----------- UI - Top Bar ----------- */
+
+main {
+  display: grid;
+  grid-template-rows: 10% 90%;
+  background-color: #000;
+}
+
+.top-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 20px;
+}
+
+.top-bar__summary {
+  display: flex;
+}
+
+.top-bar__summary span {
+  margin-right: 15px;
+}
+
+.tracking-protection {
+  display: flex;
+  align-items: center;
+}
+
+.toggle-outer {
+  position: relative;
+  height: 30px;
+  width: 60px;
+  margin-left: 10px;
+  background-color: #404850;
+  border-radius: 2px;
+}
+
+.toggle-inner {
+  position: absolute;
+  height: 80%;
+  width:45%;
+  top: 50%;
+  left: 5%;
+  transform: translateY(-50%);
+  background-color: #303539;
+  border-radius: 5px;
+}
+
+.toggle-label {
+  position: absolute;
+  top: 50%;
+  right: 5%;
+  width: 40%;
+  transform: translateY(-50%);
+  font-size: .75em;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+/* ----------- UI - Graph ----------- */
+
+.graph {
+  position: relative;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 30px;
+}
+
+.graph h1 {
+  color: #EAEAEA;
+}
+
+.graph h2 {
+  color: #73A4B8;
+}
+
+.graph__menu {
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  width: 40px;
+}
+
+.graph__menu__icon {
+    width: 90%;
+    background-color: #171E25;
+    padding: 10px;
+    border-radius: 3px 0 0 3px;
+}
+
+.graph__menu__icon:first-child {
+  margin-bottom: 5px;
+  background-color: transparent;
+}
+
+.graph__menu__icon:last-child {
+  margin-top: 5px;
+}
+
+/* ----------- UI - Graph Controls ----------- */
+
+footer {
+  grid-row-start: 2;
+  display: grid;
+  grid-template-columns: repeat(5, 20%);
+}
+
+footer h2 {
+  padding-bottom: 5px;
+  border-bottom: 1px solid #EAEAEA;
+}
+
+footer div:first-child {
+  grid-column-end: span 4;
+}
+
+.graph__controls {
+  display: grid;
+  grid-column-gap: 10px;
+  grid-template-columns: repeat(3, 170px);
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.footer-button.active {
+  background-color: #73A4B8;
+  border-top: 2px solid #6FC3E5;
+}
+
+.footer-button.active:hover {
+  border-top-color: #fff;
+}
+
+.col-1 {
```

Comment:
This seems like a copy of a grid based layout using classes. I'm not sure using these row and col classes is a good pattern here. Maybe we should discuss other solutions (right now it's late so not sure).